src/SFA.DAS.EmployerAccounts/DependencyResolution/ConfigurationRegistry.cs

```diff
@@ -13,9 +13,9 @@ public ConfigurationRegistry()
             For<EmployerFinanceConfiguration>().Use(() => ConfigurationHelper.GetConfiguration<EmployerFinanceConfiguration>("SFA.DAS.EmployerFinance")).Singleton();
             //For<EmployerAccountsReadStoreConfiguration>().Use(() => ConfigurationHelper.GetConfiguration<EmployerAccountsReadStoreConfiguration>("SFA.DAS.EmployerAccounts.ReadStore")).Singleton();
```

Comment:
Either this line should be uncommented and the hard-coded values below removed from here and added into das-employer-config, or it could use the auto-configurator.

Reply:
AutoConfig will do this

Reply:
If we use auto config, then don't we need to do something along the lines of...

```
IncludeRegistry<AutoConfigurationRegistry>();
For<EmployerAccountsReadStoreConfiguration>().Use(c => c.GetInstance<ITableStorageConfigurationService>().Get<EmployerAccountsReadStoreConfiguration>());
```
Also, `ConfigurationRegistry` is used in 5 different places. Do we need access to EmployerAccountsReadStoreConfiguration in all those places? If not, we could only make the config available when it's actually required.